Client/Clique/Modules/Login/Login.storyboard

```diff
@@ -0,0 +1,93 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Pošto se radi o present-u, a ne pushanju view controllera, fali "X" gumb na ekranu.